test/srt/test_vision_openai_server.py

```diff
@@ -289,5 +289,24 @@ def test_regex(self):
         assert isinstance(js_obj["number_of_cars"], int)
 
 
+class TestQWen2VLServer(TestOpenAIVisionServer):
+    @classmethod
+    def setUpClass(cls):
+        cls.model = "Qwen/Qwen2-VL-7B-Instruct"
```

Comment:
Does it also work correctly for video/multi image? If so, it is awesome!

Reply:
Yes, It can work correctly in video/multi image test